index.js

```diff
@@ -54,8 +70,20 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies) {
+let total = 0;
+let count = 0;
```

Comment:
We don't need this variable. We can just use the length of the movies array using `movies.length`.